crates/theme/src/styles/players.rs

```diff
@@ -139,7 +140,7 @@ impl PlayerColors {
     }
 
     pub fn color_for_participant(&self, participant_index: u32) -> PlayerColor {
-        let len = self.0.len() - 1;
+        let len = cmp::max(self.0.len() - 1, 1);
         self.0[(participant_index as usize % len) + 1]
```

Comment:
I don't think this is resilient to cases where there are 0 colors at all, should this be switched to a `.get(...).unwrap_or(...)`?